code/src/index.css

```diff
@@ -1,3 +1,8 @@
+* {
```

Comment:
With the[ GlobalStyle](https://scalablecss.com/styled-components-global-styles/), the css file can be removed completely. This one can be in the Global.js instead if you use it in your next project. :)